tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml

```
@@ -555,6 +573,16 @@ Services:
     - --event-size 1024 --max-batch-size 100 --preload 1000000 --transport-type 1 --uamqp-transport --sync --warmup 30
     TestNames:
       Python: ProcessEventsBatchTest
+  - Test: send-event-batch
+    Arguments:
+    - --body-size 1024 --batch-size 100 --warmup 2
+    TestNames:
+      JS: SendTest
+  - Test: subscribe
+    Arguments:
+    - --events 100000 --event-size 1024 --partitions 10 --max-batch-size 100 --warmup 2
```

```suggestion
    - --event-size 1024 --max-batch-size 100 --events 1000000
```

Several changes for parity with Python:

* Order parameters the same way
* Remove limit on number of partitions (I assume this means all partitions will be used)
* Do not decrease default warmup
* I assume the `events` parameter is how many events to pre-load?  If so, use 1M instead of 100k for alignment with Python.  JS might need even higher number depending on throughput.